src/api/cvc4cpp.cpp

```diff
@@ -263,13 +263,15 @@ const static std::unordered_map<Kind, CVC4::Kind, KindHashFunction> s_kinds{
     {STRING_LEQ, CVC4::Kind::STRING_LEQ},
     {STRING_PREFIX, CVC4::Kind::STRING_PREFIX},
     {STRING_SUFFIX, CVC4::Kind::STRING_SUFFIX},
+    {STRING_ISDIGIT, CVC4::Kind::STRING_ISDIGIT},
```

Comment:
Wouldn't it be more consistent to call this kind `STRING_IS_DIGIT` (to match `STRING_TO_REGEXP`/`STRNG_IN_REGEXP`?

Reply:
Sure, ill update

Reply:
We should really consider changing STRING_STRREPL -> STRING_REPLACE too then :)

Reply:
I agree, we should make everything consistent now that the standard exists (also names like `STRING_CONTAINS`, `STRING_FROM_INT`, `STRING_TO_INT`).

Reply:
Let's do this on a separate PR.